rules_building_block/discovery_signal_unusual_user_host.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2023/10/10"
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/10/10"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule leverages alert data from various Discovery building block rules to alert on signals with unusual unique
+host.id and user.id entries.
+"""
+from = "now-9m"
+index = [".alerts-security.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Unusual Discovery Activity by User"
+risk_score = 21
+rule_id = "cf575427-0839-4c69-a9e6-99fde02606f3"
+severity = "low"
+tags = ["Domain: Endpoint",
+        "OS: Windows",
+        "Use Case: Threat Detection",
+        "Tactic: Discovery",
+        "Rule Type: Higher-Order Rule"
+        ]
+timestamp_override = "event.ingested"
+building_block_type = "default"
+type = "new_terms"
+query = '''
+host.os.type:windows and event.kind:signal and kibana.alert.rule.rule_id:(
+  "d68e95ad-1c82-4074-a12a-125fe10ac8ba" or "7b8bfc26-81d2-435e-965c-d722ee397ef1" or
```

Comment:
idea:

we may want to consider adding a unit test where, when `kibana.alert.rule.rule_id` exists as a field in the query, the value rule IDs are verified to exist, to stay synced in cases of deprecations


cc: @terrancedejesus @eric-forte-elastic

Reply:
in a separate PR of course